edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Docker/SystemResourcesMetrics.cs

```diff
@@ -111,7 +111,7 @@ public void Start(ILogger logger)
         {
             if (ApiVersion.ParseVersion(this.apiVersion).Value >= ApiVersion.Version20191105.Value)
             {
-                this.updateResources = new PeriodicTask(this.Update, TimeSpan.FromMinutes(5), TimeSpan.FromMinutes(1), logger, "Get system resources", false);
+                this.updateResources = new PeriodicTask(this.Update, TimeSpan.FromSeconds(30), TimeSpan.FromMinutes(1), logger, "Get system resources", false);
```

Comment:
Why increase the frequency?

Reply:
It's much more accurate to cpu usage. every 5 minutes doesn't catch any spikes. 30 seconds still has very little performance impact and is better about spikes. This should probably be configurable, I'll add that.